lib/rules/no-arrow-function-lifecycle.js

```diff
@@ -7,26 +7,7 @@
 const Components = require('../util/Components');
 const astUtil = require('../util/ast');
 const docsUrl = require('../util/docsUrl');
-
-const lifecycleMethods = [
-  'getDefaultProps',
-  'getInitialState',
-  'getChildContext',
-  'getDerivedStateFromProps',
-  'componentWillMount',
-  'UNSAFE_componentWillMount',
-  'componentDidMount',
-  'componentWillReceiveProps',
-  'UNSAFE_componentWillReceiveProps',
-  'shouldComponentUpdate',
-  'componentWillUpdate',
-  'UNSAFE_componentWillUpdate',
-  'getSnapshotBeforeUpdate',
-  'componentDidUpdate',
-  'componentDidCatch',
-  'componentWillUnmount',
-  'render'
-];
+const lifecycleMethods = require('../util/lifecycleMethods');
```

Comment:
can we also use this list in the other places in the codebase that use lifecycle methods?

Reply:
Sure